aerogear/mobile-developer-console
# Aerogear 8137 wl

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8163

## Verification

1. Provision push, and create an app in the MDC.
2. Create an ios binding, and then an android binding.
3. Try delete the bindings. You should see there are 2 items in the delete drop down of the bound push service. You should be able to delete all of the bindings.
